prisma/schema.prisma

```diff
@@ -10,58 +10,102 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = "postgresql://user:geri.softup1@localhost:5432/mydb?secrets_management_system_dev=public"
+  url      = "postgresql://postgres:1111@localhost:5434/secrets_management_system_dev"
```

Comment:
Security issue: Do not put the database url directly in the codebase. 
Hardcoded credentials are a big NO. Even though the url is from localhost, it is best to have it separately in the environment variables  (**.env**) to keep sensitive data outside the codebase.